imu_sensor:
  ros__parameters:
    sensor_frequency: 100.0 # [Hz]
    temp_filter_alpha: 0.8 # Alpha value for temperature data
    temp_filter_beta: 0.01 # Beta value for temperature data
    imu_filter_lin_accel_alpha: 0.85 # Alpha value for linear acceleration data
    imu_filter_lin_accel_beta: 0.02 # Beta value for linear acceleration data
    imu_filter_ang_vel_alpha: 0.85 # Alpha value for angular velocity data
    imu_filter_ang_vel_beta: 0.02 # Beta value for angular velocity data
    imu_filter_orient_alpha: 0.9 # Alpha value for orientation data
    imu_filter_orient_beta: 0.01 # Beta value for orientation data
    mag_filter_alpha: 0.8 # Alpha value for magnetic field data
    mag_filter_beta: 0.01 # Beta value for magnetic field data
simulated_imu:
  ros__parameters:
    sensor_frequency: 100.0 # [Hz]